python-checks/src/test/resources/checks/tests/assertOnDissimilarTypes.py

```diff
@@ -194,13 +194,34 @@ def test_type_of_instance_and_class_object(self):
 
 
 from enum import Enum
+class ComparingTypeAndEnum(unittest.TestCase):
+    def test_type_of_enum_and_enum_class(self):
+        EnumType = Enum("EnumType", names=["one", "two"])
+        enum_member = EnumType(1)
+        # FP SONARPY-2332: EnumType is a proper "type" object as an instance of "Flag", due to it being an Enum
```

Comment:
the FP comment doesn't seem to be correct. 

```suggestion
        # FP SONARPY-2332: EnumType is a proper "type" object since the Enum constructor with argument creates a new Enum type
```